_posts/2017/enterdate-building-multiple-repository-pr-test.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+title: "Build a multiple repository pull request test"
+date: "2017-01-05 10:00:00"
+image: blog-header/challenge-cube.jpg
+categories: tech-stories
+authors: ["Florian"]
+---
+
+Did you ever face the problem of building a feature over multiple repositories? If you start testing the single repository, you may have unknown side effects when your code is rolled out. On the other hand, if you start testing the repositories together, you may have an inconsistent state of one of your code changes. This will lead to either errors in the test suite or wrong testing results. Broken tests, wrong test results and unknown side effects causes a pipeline to misses its validity.
```

Comment:
Broken tests, wrong test results and unknown side effects cause a pipeline to misses its validity.